tensorflow_hub/resolver.py

```diff
@@ -515,13 +518,27 @@ def _append_format_query(self, handle, format_query):
   def _set_url_context(self, context):
     """Add an SSLContext to support custom certificate authorities."""
     self._context = context
-
+    self._maybe_disable_cert_validation()
   def _call_urlopen(self, request):
     # Overriding this method allows setting SSL context in Python 3.
+
```

Comment:
Still needs to be removed...

Reply:
I need to push it to origin
Right now I have this changed in my remote branch